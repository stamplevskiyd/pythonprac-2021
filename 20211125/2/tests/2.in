ýòî - òåñò ñ íåñîîòâåòñòâóþùèìè ñèìâîëàìè лишние символы
