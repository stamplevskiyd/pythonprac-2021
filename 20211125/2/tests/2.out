это - тест с несоответствующими символами ?????? ???????
